test/e2e/cypress/e2e/sap_system_details.cy.js

```diff
@@ -60,23 +60,23 @@ context('SAP system details', () => {
         cy.get('@tableCell').eq(6).should('contain', instance.Status);
         cy.get('@tableCell')
           .eq(6)
-          .find('span')
-          .should('have.class', healthMap[instance.StatusBadge]);
```

Comment:
Can we remove this `StatusBadge` from the fixtures maybe?
If it is not used anymore...